tests/graphql_spec.rs

```diff
@@ -369,8 +362,8 @@ fn test_failures_in_client_sdl() -> std::io::Result<()> {
   Ok(())
 }
 
-#[test]
-fn test_merge_sdl() -> std::io::Result<()> {
+#[tokio::test]
+async fn test_merge_sdl() -> std::io::Result<()> {
```

Comment:
What is the purpose of changes in this file?

Reply:
was needed in some intermediate changes, but now is obsolete, I've removed it